doc/release/eon_1.0.0.md

```diff
@@ -0,0 +1,58 @@
+# Release version - EON 1.0.0
+
+---
```

Comment:
let's add an "Update from previous version" section with any specific instruction for the update. For this version:
- remove peers folder?
- update config file if any new parameter is different from the default